### ISSUE TITLE
Have a way to watch build progress

### ISSUE THREAD

**Issue by dwt** [zkokaja/Brewlet#22]
i.e. some way to show the text output from brew, so I can see the progress and easily inspect errors.

Maybe as simple as some apple script to open Konsole with the right log open?

**Comment by zkokaja** [comment]
Thanks for the suggestion! Currently I started writing the output to `~/Library/Logs/Brewlet/`, but adding a menu item to "Open log" that will open the file itself or maybe the console may be a good addition. I'm concerned about cluttering the interface, so I'm considering on making it a hidden menu item that you appears if you option-click the icon.

**Comment by dwt** [comment]
I would suggest making that the default behaviour of the progress indicator in the menu. It currently reads 'updating' and would be the perfect place to click on to see what is going on.

Opening the relevant log file in Konsole would probably be the easiest way to show the log output as it is being written.

**Comment by zkokaja** [comment]
Hm that's fair, 'Updating' isn't too informative for those that want to dig deeper. Currently this is what I see in my Console app (using the HEAD build) in the Log Reports section, is this what you had in mind?

![image](https://user-images.githubusercontent.com/252288/105366380-450f8380-5bcd-11eb-9fd6-81334adad0e1.png)

**Comment by dwt** [comment]
Indeed! 👍

**Comment by zkokaja** [comment]
I implemented an option that should cover this in commit 7ecd9a7. I decided to use the `option` key as a way to toggle the Updating menu item into "Open log". Tested it for a couple weeks so should be good. In the future I should consider having multiple log files, currently the same one gets overridden with every action.

**Comment by dwt** [comment]
I'll try it as soon as it is released

**Comment by dwt** [comment]
I realise that I might have been a bit of an ass requiring a release before I test this. Do you have nightly builds available somewhere, or do I have to build this myself? For brew formulas usually you can build --head yourself, but I'm not so sure for casks. is there a way?

**Comment by cmpadden** [comment]
> I realise that I might have been a bit of an ass requiring a release before I test this. Do you have nightly builds available somewhere, or do I have to build this myself? For brew formulas usually you can build --head yourself, but I'm not so sure for casks. is there a way?

I don't believe that nightly builds are being generated -- though that is a good idea. The GitHub release action is only triggered on a Git tag. If you would like to build the project yourself, you can use the following commands:

```
xcodebuild build -project 'Brewlet.xcodeproj' -scheme 'Brewlet' -destination 'generic/platform=macOS' -configuration Release CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
xcodebuild -scheme 'Brewlet' -configuration Release archive -archivePath Bl.xcarchive
xcodebuild -exportArchive -exportPath . -archivePath Bl.xcarchive/ -exportOptionsPlist Brewlet/exportOptions.plist
```

As described in the [`build-release.yml` file](https://github.com/zkokaja/Brewlet/blob/master/.github/workflows/build-release.yml#L17).

Or alternatively, you can open the project in Xcode and run/build it there.

**Comment by dwt** [comment]
Now that I've written this, of course my machine is in repair at apple for 10 days. :-/

I'll get back to this as soon as I am back at my machine - currently am stuck on an old backup machine with 10.13

**Comment by zkokaja** [comment]
Implemented in v1.6, thanks for the suggestion!
